contents/fdroid.mdx

```diff
@@ -21,3 +21,37 @@ cname: 'fdroid'
 {{http_protocol}}{{mirror}}/archive?fingerprint=43238D512C1E5EB2D6569F4A3AFBF5523418B82E0A3ED1552770ABB9A9C9CCAB
 ```
 </CodeBlock>
+
+如果你尚未安装 F-Droid 客户端，你可以直接从 [f-droid.org](https://f-droid.org) 下载。
+<details>
+<summary>我无法访问 <tt>f-droid.org</tt></summary>
+<p>对于首次安装的用户，<tt>f-droid.org</tt> 是官方下载地址，也应当始终是你的首选。如果你无法访问此网站，那么下方有其他选项。你将需要格外谨慎，确保使用你信任的来源。恶意行为者可能会将你带到有伪造及潜在恶意的 F-Droid 客户端。</p>
+<details>
+<summary>是的，我真的需要从其他来源下载。</summary>
+<p>这些替代来源由 F-Droid 团队提供，请验证它们正确：</p>
```

Comment:
I really don't like the double confirmations here. One `<details>` is already enough (I'd rather have none at all), two is too many.

Reply:
I know it is annoying, but I do believe it is important.   There is a big emphasis on steering the user to f-droid.org so it remains the one trusted source for first time installers.  Users should only ever try anything else if it is a last resort, and they know what they are doing. Then it should be something that the user can verify themselves.  We're working on simple methods for verifying and welcome any ideas or contributions there:

https://gitlab.com/fdroid/wiki/-/wikis/FAQ#how-can-i-verify-the-downloaded-f-droidapk

We're further developing this idea for the mirror index pages:
https://gitlab.com/fdroid/fdroiddata/-/merge_requests/16049

Reply:
Here's how I think about it: anyone can put instructions on a website anywhere.  They can link to _F-Droid.apk_ and say it is trustworthy.  We all know that is also abused, and malware gets distributed that way.  So it is important that users do not get used to F-Droid instructions telling them to just trust this download, unless they are on the one thing that F-Droid clearly controls: https://f-droid.org.  Sure the mirrorz mirrors are trustworthy, but can non-technical users really tell the difference between a trustworthy mirror and a malware setup made to look like a mirror?

Reply:
That said, I can simplify if that is a requirement for getting merged.

Reply:
As I noticed, fdroid has already put some warnings on [its landing page](https://mirrors.tuna.tsinghua.edu.cn/fdroid/repo/) on mirrors sites.

So maybe it's better to point users to these pages for instructions, and leaving the help page not too complex

Reply:
I reduced it to one details view.  Since the more risky links are already in `CodeBlock` and must be copy-pasted, it seems fine to have it all in one details view.